platform-upgrade

```diff
@@ -20,6 +56,7 @@ while getopts :f option; do
             ;;
         \?)
             echo "Invalid option: -$OPTARG" >&2
+            rm $cert_file
```

Comment:
Instead of adding this to all exit points, we could do this directly after creation:

```
function cleanup {
    rm "$cert_file"
}
trap cleanup EXIT
```